### DISCUSSION TITLE
Check if an element does not exist in array

### DISCUSSION THREAD

**Discussion by kartiktatiparti** [trinodb/trino#16947] [Q&A]
I have a column with arrays and value with the condition that if the value does not exist in the array, then I consider that value. Going through the documentation I found that JSON_ARRAY_CONTAINS checks if a value exists in an array. Is there any function or way to check the negative condition that the element does not exist in an array? 

I believe I can separate the query into a CTE and achieve this but wanted to know if there is a direct function.

**Comment by hashhar** [comment]
You can negate the output of the function using `NOT`. `SELECT true FROM table WHERE NOT json_array_contains('[1, 2, 3]', 2);`.
